web/app/js/components/Tap.jsx

```diff
@@ -292,7 +293,7 @@ class Tap extends React.Component {
       pendingRequests: true
     });
 
-    let url = "/api/tps-reports?resource_type=all&all_namespaces=true";
+    let url = ApiHelpers("").urlsForResource("all");
```

Comment:
Thank you for fixing these!
Actually, we can just use `this.api.urlsForResource` here since we're using withContext in this module anyway!